test/Microsoft.AspNet.Mvc.FunctionalTests/ConnegTests.cs

```diff
@@ -360,5 +360,21 @@ public class ConnegTests
             // Assert
             Assert.Equal(HttpStatusCode.NotAcceptable, response.StatusCode);
         }
+
+        [Fact]
+        public async Task ProducesAttribute_And_FormatFilterAttribute()
+        {
+            // Arrange
+            var server = TestServer.Create(_provider, _app);
+            var client = server.CreateClient();
+            var expectedContentType = MediaTypeHeaderValue.Parse("application/json");
+
+            // Act
+            var response = await client.GetAsync("http://localhost/ProducesContentBase/MethodWithFormatFilter.json");
+
+            // Assert
+            Assert.Equal(expectedContentType, response.Content.Headers.ContentType);
+            var body = await response.Content.ReadAsStringAsync();            
```

Comment:
all Conneg tests were WIP when you saw these, I just pushed those I can go thru some other things with Ryan.